datafusion/core/tests/sqllogictests/src/main.rs

```diff
@@ -130,12 +130,6 @@ async fn context_for_test_file(file_name: &str) -> SessionContext {
             setup::register_aggregate_tables(&ctx).await;
             ctx
         }
-        "information_schema.slt" => {
```

Comment:
since this is now controllable dynamically, we do not have to hard code the sqltest runner